scripts/styles.sh

```diff
@@ -6,14 +6,25 @@ OUTPUT_STYLECSS=internal/view/assets/css/style.css
 INPUT_ARCHIVECSS=internal/view/assets/less/archive.less
 OUTPUT_ARCHIVECSS=internal/view/assets/css/archive.css
 
+# Detect support of avx2
+
+if [ -x "$(command -v bun)" ]; then
```

Comment:
I believe it's easier to just set `BUN=bun` by default and then check support for `avx2` modifying the command:

```sh
BUN=bun
if ! rep -q avx2 /proc/cpuinfo; then
    BUN="sde -chip-check-disable -- bun"
    echo "Your CPU does not support avx2 so we use sde, for more information please look at https://github.com/oven-sh/bun/issues/762#issuecomment-1186505847"
fi
```

Reply:
Sadly I don't have access to a Intel Mac, but ignoring that clause (because macos does not have `/proc/cpuinfo`) works well on mac ARMs. See this patch and let me know if it works on your machine
[macos.patch](https://github.com/go-shiori/shiori/files/15446921/macos.patch)